### PR TITLE
add real-time-apps-with-go-and-reactjs part 3

### PR DESCRIPTION
**Pull request by schaeferthomas**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Thomas Schäfer github@tschaefer.eu

### CODE REVIEW

File: tutorials/real-time-apps-with-go-and-reactjs/03.en.md [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+  - Holds the state and handles server events
+  - Passes `chatLines` down to `<ChatLineHolder>` and `onSend` down to `<ChatInput>`
+  - The `EventSource` class is used to subscribe to events.
+  - A simple `fetch` call sends a message
+
+- `<ChatLineHolder>` - Receives multiple chat lines and renders `<ChatLine>` components
+-  `<ChatLine>` - Renders a single chat message
+- `<ChatInput>` - Holds the text a users types in local state. Calls `onSend` when the `send` button is clicked
+
+## Thanks for Reading
```
Comment (n-se) [inline tutorials/real-time-apps-with-go-and-reactjs/03.en.md]: Please add the MIT license block at the bottom and sign it.

File: tutorials/real-time-apps-with-go-and-reactjs/03.en.md [hunk 2]
```diff
@@ -0,0 +1,431 @@
...
+date: "2020-01-16"
+title: "Creating a Basic UI with ReactJS | Real-time Apps with Go and ReactJS"
+short_description: "In this part, we will write a very simple chat application using ReactJS and the SSE server that was implemented in the last part.  We will build a live dashboard that monitors servers and receives webhooks via GitlabCI (or any CI really), rendering this data live to every client that is connected without any delay."
+tags: ["Development", "Lang:JS"]
+author: "Thomas Schäfer"
+author_link: "https://github.com/schaeferthomas"
+author_description: "My life for Aiur"
+language: "en"
+available_languages: ["en"]
+header_img: "https://public.headt.rip/github/hetzner/realtime-tutorial/header.png"
```
Comment (n-se) [inline tutorials/real-time-apps-with-go-and-reactjs/03.en.md]: External resources do not work here.

File: tutorials/real-time-apps-with-go-and-reactjs/03.en.md (hunk 1 above)
Comment (schaeferthomas) [reply tutorials/real-time-apps-with-go-and-reactjs/03.en.md]: Block has been added and signed off

File: tutorials/real-time-apps-with-go-and-reactjs/03.en.md (hunk 2 above)
Comment (schaeferthomas) [reply tutorials/real-time-apps-with-go-and-reactjs/03.en.md]: Header image has been removed

**Pull request merged**